### ISSUE TITLE
Multiple saved same payment instrument

### ISSUE THREAD

**Issue by kourosko** [Rebilly/Rebilly-NET-SDK#2]
If i'll use rebilly.js and tokenization for creating payment card instrument i can save multiple times the same card with the same values. This should't be happening. i should have unique cards only. Am i wrong?

**Comment by adamaltman** [comment]
Yes, you can.  It is fine for it to happen.  There is a difference between the payment token and the payment instrument.  The payment token can be created without much permissions direct from the client's browser.  A payment token will expire after a day, if it's not converted to the appropriate instrument, like a payment card, let's say, for this example.

There is no restriction for creating duplicate payment cards, although a customer can only have one default payment instrument used for autopay.

**Comment by kourosko** [comment]
I have created the same payment instrument multiple times with different token for the same customer. that is what i'm telling. It shouldn't be happening. It should replace the older card with the new one duplicate.

**Comment by adamaltman** [comment]
Yes, this is the expected functionality.
